src/csv/manifest/manifest.go

```diff
@@ -3,14 +3,15 @@ package manifest
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/iancoleman/orderedmap"
 	"keboola.processor-split-table/src/kbc"
 	"keboola.processor-split-table/src/utils"
 	"os"
 )
 
 type Manifest struct {
 	path    string
-	content map[string]interface{} // decoded JSON content
+	content *orderedmap.OrderedMap // decoded JSON content
```

Comment:
Default map implementacia nezohladnuje poradie, co robilo problem v testoch. JSONu je jedno v akom poradi su kluce, ale v testoch to chceme mat rovnako. Preto som pouzil inu implementaciu, ktora poradie zohladnuje.